site/downloads.markdown

```diff
@@ -25,152 +22,70 @@ This page describes the installation of the Haskell toolchain, which consists of
 
 * * *
 
-## Alternative installation options { #ghc-install-manual }
-
-*   [Using a package manager](#package-manager)
-
-*   [Official bindists](#bindists)
-
-*   [Building from source](#from-source)
-
-*   [Other options](#other-options)
-
-### Using a package manager { #package-manager }
-
-#### Linux
-
-Refer to your distribution package manager documentation. For convenience, below are a few distribution specific instructions, outlining 3rd party repository use as well.
-
-*Note: Most Linux distros don't allow different versions of GHC to be installed in parallel. Sometimes the GHC packages are a bit outdated as well. In either case, consider using [GHCup](#ghcup) instead.*
-
-*Note: haskell-language-server isn't packaged by most distributions. Instead you will have to install it manually, see the [release page](https://github.com/haskell/haskell-language-server/releases), unless you use VSCode, which bootstraps everything automatically. This is optional.*
+### Native OS packages { #distro-packages }
```

Comment:
It would probably be clarifying to provide a short introduction to this section, perhaps something like

> If you prefer to install the Haskell toolchain through your operating system's native package management system then you might find the links below useful. These installation methods are not supported by the Haskell community.